db-helpers.js

```diff
@@ -103,24 +105,11 @@ function getUsers (callback) {
     })
 }
 
-function getUserByEmail (email, callback) {
+function getFormSubmission (email, dateSubmit, callback) {
     openDbConnection( (err, client, done) => {
-        client.query('SELECT * FROM users WHERE email = $1', [email], function (err, result) {
```

Comment:
Just curious why email is not enough? Would there be any case where the `date_submit` could be lost?